manim/animation/transform.py

```diff
@@ -238,6 +262,30 @@ def __init__(
 
 
 class CounterclockwiseTransform(Transform):
+    """Transforms Mobjects while moving them in Counter-Clockwise direction.
+
+    Examples
+    --------
+
+    .. manim:: CounterclockwiseTransform1
+        :quality: medium
+
+        class CounterclockwiseTransform1(Scene):
+            def construct(self):
+                self.play(CounterclockwiseTransform(Square(), Circle()))
+
+    .. manim:: CounterclockwiseTransform2
+        :quality: medium
+
+        class CounterclockwiseTransform2(Scene):
+            def construct(self):
+                t1 = Text("1").shift(UP)
+                t2 = Text("2").shift(DOWN)
+                self.add(t1, t2)
+                self.play(CounterclockwiseTransform(t1, t2))
```

Comment:
The same comment as for `ClockwiseTransform2` applies here as well.

Reply:
i'll change it